### PR TITLE
refactor(docs-infra): clean up CSS styles

### PR DESCRIPTION
**Pull request by gkalpak**
This PR contains several refactoring commits that clean up our angular.io styles. All commits are refactorings (i.e. no feature/fix), so there should be no visual difference in the built app. (If there is, that is a mistake 😱)

See the individual commits for details. 

NOTES:
- Most commits are easier to review with whitespace changes ignored. (In GitHub, this can be achieved by appending `?w=1` to the URL.)
- I have tried to limit changes to a signle file/component in each commit (to keep them easier to review).
- There are three main types of clean-ups:
    - Removing unused styles.
    - Merging blocks that target the same elements.
    - Scope styles to specific components/elements.

### CODE REVIEW

**Review by petebacondarwin — APPROVED** [review]
Nice tidy up. A couple of comments but otherwise LGTM.

File: aio/src/styles/1-layouts/_sidenav.scss [hunk 1]
```diff
@@ -73,119 +55,139 @@ mat-sidenav-container.sidenav-container {
...
+    margin: 0;
   }
 
-  //icons _within_ nav
-  .mat-icon {
-    height: 24px;
-    width: 24px;
-  }
-}
+  aio-nav-item {
```
[the remark this answers is not in the document]
Comment (gkalpak) [reply aio/src/styles/1-layouts/_sidenav.scss:75]: It actually expands to `aio-nav-menu aio-nav-item .vertical-menu-item`. Are you suggesting to change it to `aio-nav-menu .vertical-menu-item`?

File: aio/src/styles/1-layouts/_sidenav.scss [hunk 2]
```diff
@@ -45,15 +32,45 @@ mat-sidenav-container.sidenav-container {
...
     max-width: 82%;
   }
-}
 
-mat-sidenav-container.sidenav-container.mat-drawer-container.mat-sidenav-container,
-mat-sidenav-container .sidenav-content {
-  background-color: $white;
-}
+  .sidenav-content {
+    background-color: $white;
```
[the remark this answers is not in the document]
Comment (gkalpak) [reply aio/src/styles/1-layouts/_sidenav.scss]: Removed.

File: aio/src/styles/1-layouts/_sidenav.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (petebacondarwin) [reply aio/src/styles/1-layouts/_sidenav.scss:74]: My mistake. Carry on

**Pull request closed without merging**